bin/varnishtest/tests/v00066.vtc

```diff
@@ -0,0 +1,22 @@
+varnishtest "long string coverage"
+
+varnish v1 -vcl {
+	backend default none;
+
+	sub vcl_recv {
+		return (synth(200));
+	}
+
+	sub vcl_synth {
+                set resp.body = """{"key":"value"}""";
```

Comment:
this needs more `\n` and `{"`

Reply:
This is already tested in the built-in VCL for {", if you think a different string for the test would be better please let me know what that string would be.

Reply:
```
std.log("""
{" "} "  
""");
```

would work for me, but `varnishtest` may not cooperate.

Reply:
If you have a `{...}` balance issue in `varnishtest` multi-line strings you can often work around it with VCL:

```
varnish v1 -vcl+backend {
    import std;
    sub vcl_recv {
        std.log("{");
        # adding '}' for balance
    }
}
```

Add the right brace at the right location in a comment.